flink-ml-benchmark/src/test/java/org/apache/flink/ml/benchmark/BenchmarkTest.java

```diff
@@ -54,6 +60,44 @@ public void testParseJsonFile() throws Exception {
         assertTrue(benchmarks.containsKey("KMeansModel-1"));
     }
 
+    @Test
+    public void testJsonFileLegality() throws IOException, ClassNotFoundException {
+        File resourcesDir =
```

Comment:
Can you help update `testParseJsonFile()` to also use this approach instead of having to copy the file to `test-conf.json`?

Reply:
Sure.